recipe/set_bld_opts.bat

```diff
@@ -66,6 +73,11 @@ set BLD_OPTS=%WIN64% ^
     GEOTIFF_INC="-I%LIBRARY_INC%" ^
     GEOTIFF_LIB=%LIBRARY_LIB%\geotiff_i.lib ^
     LIBICONV_INCLUDE="-I%LIBRARY_INC% -DICONV_CONST= " ^
-    LIBICONV_LIBRARY="%LIBRARY_LIB%\iconv.lib"
+    LIBICONV_LIBRARY="%LIBRARY_LIB%\iconv.lib" ^
+    POPPLER_ENABLED=YES ^
+    POPPLER_CFLAGS="-I%LIBRARY_INC%\poppler -DPOPPLER_0_20_OR_LATER= -DPOPPLER_0_23_OR_LATER= -DPOPPLER_0_58_OR_LATER= -DPOPPLER_HAS_OPTCONTENT= -DPOPPLER_BASE_STREAM_HAS_TWO_ARGS= " ^
+    POPPLER_MAJOR_VERSION=%POPPLER_MAJOR_VERSION% ^
+    POPPLER_MINOR_VERSION=%POPPLER_MINOR_VERSION% ^
+    POPPLER_LIBS="%LIBRARY_LIB%\poppler.lib %LIBRARY_LIB%\freetype.lib advapi32.lib"
```

Comment:
I have to link to the static lib on Windows as not all the needed symbols are `export`ed by `poppler`. This means I have to list any other needed libs that aren't already linked to `gdal`...

Reply:
Update: managed to enable shared libs in https://github.com/conda-forge/poppler-feedstock/pull/37. Seems to link ok now.